packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -260,7 +260,8 @@ let interruptedBy: Fiber | null = null;
 
 // Do not decrement interaction counts in the event of suspense timeouts.
 // This would lead to prematurely calling the interaction-complete hook.
-let suspenseDidTimeout: boolean = false;
+// Instead we wait until the suspended promise has resolved.
+let interactionsHaveBeenSuspended: boolean = false;
```

Comment:
Suggestion: `nextRenderIncludesTimedOutPlaceholder`

Reply:
This name isn't accurate for the initial `null` render but I don't care 😁